src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.diff.sql

```diff
@@ -29,22 +23,143 @@ CREATE UNIQUE NONCLUSTERED INDEX IX_CustomTag_TagPath ON dbo.CustomTag
+CREATE TABLE dbo.CustomTagDouble (
+    TagKey                  BIGINT               NOT NULL, --PK
+    TagValue                FLOAT(53)            NOT NULL,
+    StudyKey                BIGINT               NOT NULL, --FK
+    SeriesKey               BIGINT               NULL,     --FK
+    InstanceKey             BIGINT               NULL,     --FK
+    Watermark               BIGINT               NOT NULL
+) WITH (DATA_COMPRESSION = PAGE)
+
+CREATE UNIQUE CLUSTERED INDEX IXC_CustomTagDouble ON dbo.CustomTagDouble
+(
+    TagKey,
+    TagValue,
+    StudyKey,
+    SeriesKey,
+    InstanceKey
+)
+
+/*************************************************************
+    Custom Tag Data Table for VR Types mapping to DateTime
+    Note: Watermark is primarily used while re-indexing to determine which TagValue is latest.
+          For example, with multiple instances in a series, while indexing a series level tag,
+          the Watermark is used to ensure that if there are different values between instances,
+          the value on the instance with the highest watermark wins.
+**************************************************************/
+CREATE TABLE dbo.CustomTagDateTime (
+    TagKey                  BIGINT               NOT NULL, --PK
+    TagValue                DATETIME2(7)         NOT NULL,
+    StudyKey                BIGINT               NOT NULL, --FK
+    SeriesKey               BIGINT               NULL,     --FK
+    InstanceKey             BIGINT               NULL,     --FK
+    Watermark               BIGINT               NOT NULL
+) WITH (DATA_COMPRESSION = PAGE)
+
+CREATE UNIQUE CLUSTERED INDEX IXC_CustomTagDateTime ON dbo.CustomTagDateTime
+(
+    TagKey,
+    TagValue,
+    StudyKey,
+    SeriesKey,
+    InstanceKey
+)
+
+/*************************************************************
+    Custom Tag Data Table for VR Types mapping to PersonName
+    Note: Watermark is primarily used while re-indexing to determine which TagValue is latest.
+          For example, with multiple instances in a series, while indexing a series level tag,
+          the Watermark is used to ensure that if there are different values between instances,
+          the value on the instance with the highest watermark wins.
+	Note: The primary key is designed on the assumption that tags only occur once in an instance.
+**************************************************************/
+CREATE TABLE dbo.CustomTagPersonName (
+    TagKey                  BIGINT               NOT NULL, --FK
+    TagValue                NVARCHAR(200)        COLLATE SQL_Latin1_General_CP1_CI_AI NOT NULL,
+    StudyKey                BIGINT               NOT NULL, --FK
+    SeriesKey               BIGINT               NULL,     --FK
+    InstanceKey             BIGINT               NULL,     --FK
+    Watermark               BIGINT               NOT NULL,
+	WatermarkAndTagKey      AS CONCAT(TagKey, '.', Watermark), --PK
```

Comment:
nit: this formatting is off slightly

Reply:
I guess it's problem with github, it looks good on VS
![image](https://user-images.githubusercontent.com/67345894/108402575-93b33c00-71d2-11eb-9c25-d8ee3b65ecee.png)

Reply:
You used a tab there instead of spaces. You can show white space in VS to help see it